tests/yast2_gui/yast2_network_settings.pm

```diff
@@ -64,7 +70,9 @@ sub run {
 
     #	Save network setting and it can take long time, exit
     send_key "alt-o";
-    wait_still_screen(60);
+    sub post_run_hook {
+        assert_screen('generic-desktop', 300);
+    }
```

Comment:
syntax wrong. That has to be a sub definition outside the run method. I doubt this has worked for you